profiler/src/profile_grouped_gemm_fixed_nk.cpp

```diff
@@ -83,7 +81,7 @@ int profile_grouped_gemm_fixed_nk(int argc, char* argv[])
     const auto StrideAs = argToIntArray(argv[11]);
     const auto StrideBs = argToIntArray(argv[12]);
     const auto StrideCs = argToIntArray(argv[13]);
```

Comment:
Could it be unified with profile_grouped_gemm?

Reply:
No, because of uniquness of this operator and it's API.